lib/KIcon/iconDefinitions.js

```diff
@@ -29,6 +29,9 @@ const KolibriIcons = {
   dropdown: {
     icon: require('./precompiled-icons/material-icons/arrow_drop_down/baseline.vue').default,
   },
+  dropup: {
+    icon: require('./precompiled-icons/material-icons/arrow_drop_up/baseline.vue').default,
```

Comment:
The path here needs to be `precompiled-icons/le/dropup.vue` as that's the path where the `.vue` file was generated for the svg you added

Reply:
done, made the changes.